### ISSUE TITLE
pipenv (or possibly phonenumberlite?) doesn't install required? phonenumbers library

### ISSUE THREAD

**Issue by yyhmsg** [stefanfoulis/django-phonenumber-field#455]
With pipenv install django-phonenumber-field[phonenumberlite] - upon trying to use the module I got a ModuleNotFoundError:  No module named 'phonenumbers'

Doing pipenv install phonenumbers took that error away and all works.

I don't know if this is an issue with pipenv or with using the phonenumberlite variable.  But I didn't seen this requirement in the docs anywhere.  Thought I would post in case anyone else gets stuck.

**Comment by francoisfreitag** [comment]
Did you mean to install `django-phonenumber-field[phonenumberslite]`? (notice the `s` in phonenumber**s**lite)

**Comment by yyhmsg** [comment]
Sorry for the late response.  Yes, looking through my history I did install:
django-phonenumber-field = {extras = ["phonenumberlite"], version = "*"}

(without the "s") - but after I installed pipenv install phonenumbers everything started working so I guess I never looked back after that.  So I guess I'm just running the full version? and and can try removing phonenumbers from my pip file and maybe reinstall django-phonenumber-field correctly (I don't think that would disrupt my current models/data).

The error I got didn't say anything about phonenumberlite not being a thing, rather it said "no module named phonenumbers" - so I thought it must have been a dependency I was missing.  Sorry for the misunderstanding.
